server/controllers/routes/user/cohort/getSpecificCohort.js

```diff
@@ -0,0 +1,21 @@
+const {
+  cohort: { getSpecificCohort },
+} = require('../../../../database/queries');
+
+exports.getSpecificCohort = async (req, res, next) => {
+  try {
+    const { cohortid } = req.params;
+    const { rows } = await getSpecificCohort(cohortid);
+    const data = { ...rows[0] };
+    if (data.id) {
+      res.status(200).json({ statusCode: 200, data });
```

Comment:
status code by default will be 200
```suggestion
      res.json({ statusCode: 200, data });
```